website/docs/components/tabs/partials/guidelines/guidelines.md

```diff
@@ -47,6 +47,12 @@ Don’t trigger a Tab change via an external action, such as a submit or next bu
 ![Tabs with external trigger](/assets/components/tabs/tabs-external-trigger.png)
 !!!
 
+## Sizes
```

Comment:
In general, I think we can expand on this a little bit and be slightly more prescriptive around where we expect the different sizes to be used. A couple of things that I might consider:

- I think you hit the nail on the head with hierarchical distinction, can this come to the forefront a bit? This was the main problem the additional size was aiming to solve.
- Would be good to include something about nesting, since the large variant sits higher in the hierarchy, this could be a more effective way of communicating nested/dependent content; or, if this is something we recommend against (not mixing sizes), why, and some other solutions
- There's already a bit of content around "contained" tabs that might be useful to consider as well, _in general_ it seems like the large version should live more at the page level and the medium version should live in a contained environment a bit better.

Initial thoughts here, happy to pair on this next week to brainstorm some additional content!